Resources/Prototypes/Entities/Structures/Piping/Atmospherics/pipes.yml

```
@@ -152,6 +157,7 @@
     - 0,0,0,2
   - type: MeleeWeapon
     attackRate: 1
+    wideAnimationRotation: 1 # Delta-V
```

nit: Don't need the dash.

```suggestion
    wideAnimationRotation: 1 # DeltaV
```